webrender_traits/src/channel.rs

```diff
@@ -0,0 +1,148 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use serde::{Deserialize, Serialize};
+use std::io::Error;
+
+#[cfg(feature = "ipc")]
```

Comment:
given that everything in this module is split between `ipc` and non-ipc, I'd rather see separate files `channel.rs` and `channel_ipc.rs` with a single switch upon including them.